### PR TITLE
End nether metals

### PR DESCRIPTION
**Pull request by Biggbosscurser**
Besides couple of texture missing End metals and Nether metals mods are done.

### CODE REVIEW

**Review by zISKo666 — COMMENTED** [review]
Well, I miss the cracks and shadows. See my examples, please.

nether_diamond_ore
end_diamond_ore

**Comment by Biggbosscurser** [comment]
Yes, i got some comments and suggestions on how to do the shadows and cracks. This request can be marked as awaiting for changes. I will fix the textures when i can do the time.

File: End_Metals/mod.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "mod_id": "EndMetals",
+  "mod_name": "End_Metals",
+  "mod_dir": "/end_metals/",
```
Comment (alfw) [inline End_Metals/mod.json]: This  need to match the folder name itself. ``"mod_dir": "/End_Metals/"``

File: Nether_Metals/mod.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "mod_id": "NetherMetals",
+  "mod_name": "Nether_Metals",
+  "mod_dir": "/nether_metals/",
```
Comment (alfw) [inline Nether_Metals/mod.json]: This need to match the folder name itself. "mod_dir": "/Nether_Metals/"

**Review by alfw — CHANGES_REQUESTED** [review]
The ``mod.json`` file format is importen for our scripts to run right. Those changes needed to be made.

File: End_Metals/mod.json (hunk 1 above)
Comment (mgdgfbhevdyeh) [reply End_Metals/mod.json]: @Biggbosscurser change,please.

**Comment by alfw** [comment]
Still waiting for those changes @Biggbosscurser !

**Comment by Biggbosscurser** [comment]
I am so sorry guys I've been crazy busy lately and totally forgot about these.I will get back to it and hopefully done them before midnight.

File: Nether_Metals/mod.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "mod_id": "NetherMetals",
   "mod_name": "Nether_Metals",
-  "mod_dir": "/nether_metals/",
+  "mod_dir": "/Nether_metals/",
```
Comment (alfw) [inline Nether_Metals/mod.json]: Hey, this need to be `Nether_Metals`.

**Review by alfw — APPROVED** [review]

**Pull request merged**